charts/self-hosted/values.yaml

```
@@ -13,11 +13,12 @@ global:
     rootKey: 997ec52a856f0911442d1fccc308a408
 
   directory:
-    version: 0.21.13
+    version: 0.21.15
     rootKey: c96fe48da0735cf77fdba50134f52c43
 
   postgresql:
     localServerEnabled: true
+    existingSecret: "{{ .Release.Name }}-postgresql"
     auth:
```

This is great and certainly accomplishes the goal of a passing a postgres password via environment variable!

It's also a bit confusing that the secret key now defaults `postgres-password`. Someone would need to read the chart or look up the bitnami chart to figure out the name of the key.

I would at least document in the `README` or a comment in `values.yaml` that the `existingSecret` should contain a key `postgres-password` for the password since that's not obvious from the value keys . Better yet, add the passthrough `bitnami` configuration setting for `auth.secretKeys.adminPasswordKey` and set the default to `postgres-password`. This also make it clear that `aserto-directory` must be an admin of the database.

```suggestion
    auth:
      secretKeys:
        adminPasswordKey: postgres-password
```

---

Another great reason for this scheme is better compatibility with things like [zolando postgres-operator](https://github.com/zalando/postgres-operator), which creates a k8s secret with the key `password` by default.